app/bundles/CoreBundle/Assets/css/libraries/bootstrap/less/alerts.less

```diff
@@ -67,5 +67,9 @@
   .alert-variant(@alert-danger-bg; @alert-danger-border; @alert-danger-text);
 }
 .alert-mautic {
+  .media-body > div > a { 
```

Comment:
This doesn't seem to affect the link in the update notification box if thinking of the same thing.  Maybe change this line to `.alert-link {` ?